src/bot.ts

```diff
@@ -0,0 +1,236 @@
+import crypto from 'crypto'
+import fs from 'fs'
+import path from 'path'
+
+import {
+  BaseServiceV2,
+  ExpressRouter,
+  validators,
+} from '@eth-optimism/common-ts'
+import { Octokit } from 'octokit'
+import extract from 'extract-zip'
+import uuid from 'uuid'
+
+import { version } from '../package.json'
+
+type TOptions = {
+  secret: string
+  pat: string
+  tempdir: string
+}
+
+type TMetrics = {}
```

Comment:
What metrics would be useful here? We already get server metrics by default.

Reply:
I can't think of anything